decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -50,6 +50,14 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_meetings.content_security_handlers" do |_app|
+        Decidim.configure do |config|
+          config.content_security_policies_extra.deep_merge!({
+                                                               "frame-src" => %w(player.twitch.tv www.youtube-nocookie.com meet.jit.si)
```

Comment:
Can you add vimeo too 🙏🏽 ? 

It's a service supported too: https://github.com/decidim/decidim/blob/21bf844a88e2856e6dc9e242884ecc5ede88b69f/decidim-core/app/packs/src/decidim/editor/extensions/video_embed/index.js#LL64C29-L64C45 

```suggestion
                                                               "frame-src" => %w(player.twitch.tv www.youtube-nocookie.com meet.jit.si player.vimeo.com)
```

Reply:
@andreslucena, i had to dismiss the code suggestion, as vimeo and youtube needs to be in the core.